### DISCUSSION TITLE
something wrong of my QC results

### DISCUSSION THREAD

**Discussion by qiudashuai1** [CoBrALab/RABIES#391] [General]
I use the docker to run my data. But I don't know if it's my data problem, the QC results are very wired.
This is the  visilization of my data, if the orientation of my data is wrong?
![image](https://github.com/user-attachments/assets/fa829239-04d8-4519-a0b0-ade5d7b997ac)
This is the json file of my data:
[sub-sd1_acq-RARE_T2w.json](https://github.com/user-attachments/files/17510528/sub-sd1_acq-RARE_T2w.json)
Then is the preprocess.log file:
 [rabies_preprocess.log](https://github.com/user-attachments/files/17510447/rabies_preprocess.log)
Under files are QC report:
The first QC picture seems the matching of the upper and lower images are reversed.
![_registration](https://github.com/user-attachments/assets/f8df3960-1c89-42d0-bddf-08d6908a7681)
![sub-sd1_acq-RARE_T2w_inho_cor](https://github.com/user-attachments/assets/8053338d-c3e1-4877-ac3b-981f5bc4ca49)
![sub-sd1_task-rest_acq-EPI_run-1_bold_inho_cor](https://github.com/user-attachments/assets/7aabf80d-66f1-4fb7-8f39-45fbf6c85063)
![sub-sd1_task-rest_acq-EPI_run-1_bold_registration](https://github.com/user-attachments/assets/325cdfb2-d6de-47a6-9ab7-6c3e52df9be9)

**Comment by Gab-D-G** [comment]
Hi, I can see indeed from the RABIES QC reports that your image orientation will not match the template in the pipeline. I cannot tell from your interface though what exact orientation your images are in to begin with. You should match the RAS conventional orientation for everything to work. Perhaps you can manually infer which axes need to be fixed on your end.

**Reply by qiudashuai1** [reply]
Thanks, I have rotated the data using SPM.

**Reply by gdevenyi** [reply]
We would instead recommend that you convert from the (presumably bruker raw data) using brkraw, a known reliable conversion tool.

**Reply by qiudashuai1** [reply]
Thanks for your advise, but unfortunately, we do not have bruker raw data, only have nifti data.

**Reply by gdevenyi** [reply]
In that case our minimal suggestion is to verify the anatomical directions indicated in itksnap (A,P,L,R) correspond to the correct directions in the nifti images when loading them. I would be wary of getting L-R correct, our suggestion here is to scan a subject with reference vitamin-E tablet on a  known side to confirm orientations of data.

**Reply by qiudashuai1** [reply]
Ok, thanks, I will check it in itksnap. By the way, I have a question regarding atlas: RABIES using DSURQE atlas as default atlas, but I can't open the link of https://wiki.mouseimaging.ca/display/MICePub/Mouse+Brain+Atlases. I want to know the brain atlas labels of DSURQE, can you provide me a csv file or alternative link? Another question is, if I want to use Allen CCFv3 atlas, do I need another transform?

**Reply by Gab-D-G** [reply]
for some reason I also cannot currently access the original database of the DSURQE atlas, perhaps there are issues with the platform for now. You may be able to find the files already downloaded inside the RABIES container under $HOME/.local/share/rabies. Otherwise, if you want to use Allen atlas, you need to provide a full set of atlas files that RABIES needs as inputs (template, masks, labels...). If you're missing some masks, you can provide a brain mask instead and it will run, but just don't use whatever confound correction/analysis which depend on those masks. If you provide all files, you do not need to provide additional transforms.
